db/data_migration/20150407095340_convert_policies_to_html_publications.rb

```diff
@@ -0,0 +1,143 @@
+module PolicyMigrationHelpers
+  def self.build_html_attachment_body(policy, url_maker, alternative_format_provider)
+    callout = %|
+$CTA
+This is a copy of a document that stated a policy of the 2010 to 2015 Conservative and Liberal Democrat coalition government.
+The previous URL of this page was [#{url_maker.public_document_url(policy)}](#{url_maker.public_document_path(policy)})
+Current policies can be found at the GOV.UK [policies list](/government/policies).
```

Comment:
Apparently with this exact case in mind https://github.com/alphagov/whitehall/blob/master/test/unit/data_hygiene/govspeak_link_validator_test.rb#L47-L50